app/routes/data-quality.ts

```diff
@@ -1,3 +1,197 @@
+        ((getCount(totalCountAgendaItemsWithTitleAndDescription) || 0) /
+          (totalCountAgendaItems || 1)) *
+          100
+      );
+
+      const totalCountAgendaItemsTreated: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
+        await this.store.query('agenda-item', {
+          filter: {
+            ':or:': {
+              sessions: {
+                ':has:started-at': true,
+                ':has:ended-at': true,
+              },
+            },
+            sessions: {
+              'governing-body': {
+                'administrative-unit': {
+                  location: {
+                    label: params.municipalityLabel || undefined,
+                  },
+                },
+              },
+            },
+          },
+          size: 1,
+        });
+
+      const percentageAgendaItemsTreatedToTotal = Math.round(
+        ((getCount(totalCountAgendaItemsTreated) || 0) /
+          (totalCountAgendaItems || 1)) *
+          100
+      );
+
+      // get all classifications (governing bodies) from the store and dont get duplicates
+      const allGoverningBodyClassifications = await this.store.findAll(
+        'governing-body-classification-code'
+      );
+
+      // put all of the classifications through a loop to get the count of agenda items per classification
+
+      const agendaItemsPromises = allGoverningBodyClassifications
+        .filter(
+          // filter out duplicates
+          (classification, index, self) =>
+            self.findIndex((c) => c.label === classification.label) === index
+        )
+        .map(async (governingBody) => {
+          const agendaItems: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
+            await this.store.query('agenda-item', {
+              filter: {
+                sessions: {
+                  'governing-body': {
+                    'administrative-unit': {
+                      location: {
+                        label: params.municipalityLabel || undefined,
+                      },
+                    },
+                    classification: {
+                      label: governingBody.label,
+                    },
+                  },
+                },
+              },
+              size: 1,
+            });
+
+          return {
+            classification: governingBody.label,
+            count: getCount(agendaItems) || 0,
+          };
+        });
+
+      // resolve the promises
+      agendaItemsPerGoverningBodyClassification = await Promise.all(
+        agendaItemsPromises
+      );
+
+      agendaItemsPerGoverningBodyClassification =
+        agendaItemsPerGoverningBodyClassification.map((item) => ({
+          ...item,
+          percentage: ((item.count || 0) / (totalCountAgendaItems || 1)) * 50,
+        }));
+
+      agendaItemsPerGoverningBodyClassification.push({
+        classification: 'Totaal',
+        count: totalCountAgendaItems || 0,
+        percentage: 50,
```

Comment:
On smaller screens the graph gets buggy if it is at 100% width so I hardcode it at 50% so it's pretty much always clearly visible on classifications with long string lengths

Reply:
I prefer to keep data meaningful, and adapt data at the last place when it's needed. In other words, keep `percentage: 100` in data and divide the percentage in the template with a comment that explain why it's divided in 2. It will help the next developer or you in few weeks ;)